airbyte-integrations/connectors/source-airtable/unit_tests/test_source.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import MagicMock, patch
+
+from airbyte_cdk.models import AirbyteCatalog, ConnectorSpecification
+from source_airtable.source import SourceAirtable
+
+
+def test_spec():
+    source = SourceAirtable()
+    logger_mock = MagicMock()
+    spec = source.spec(logger_mock)
+    assert isinstance(spec, ConnectorSpecification)
+
+
+@patch("requests.get")
+def test_discover(test_config):
+    source = SourceAirtable()
+    logger_mock = MagicMock()
+    response = source.discover(logger_mock, test_config)
```

Comment:
```suggestion
    airbyte_catalog = source.discover(logger_mock, test_config)
```

Reply:
I think you could add more assertions: e.g check the streams in the catalog match the tables defined in the config, or mock helpers function and check they've been called etc.